website/content/docs/upgrading/deduplication/entity-group.mdx

```diff
@@ -0,0 +1,128 @@
+---
+layout: docs
+page_title: Fix entity and group duplicates
+description: >-
+  Fix duplicate identities for Vault entities and groups
+---
+
+# Fix entity and group duplicates
+
+Fix duplicate identities for entities and groups targeted for renaming.
+
+<Tip title="Assumptions">
+
+- You are running Vault 1.19 or later.
+- You have [deduplication **renaming** targets in your system logs](/vault/docs/upgrading/identity-deduplication).
+- You have admin permission on the relevant Vault server or cluster.
+
+</Tip>
+
+
+## Why duplicates happen
+
+Historical bugs in Vault allowed authN mounts to create duplicate entries of the
+same entity or group name if the names used difference cases. By default, Vault
+uses case-insensitive matching for names and considers `bob` and `BOB` to be the
+same name. But, past bugs have allowed for identical or different-case
+duplicates in entity and group names.
+
+Duplicate entity and group names can cause unexpected behavior and make working
+with Vault more difficult:
+
+- Fetching entities/groups by name returns the first identity found. Depending
+  on your Vault version, the identity returned can vary depending on the server
+  handling the request and when the last seal/unseal event occurred.
+- Deleting entities/groups by name is not guaranteed to delete all duplicates.
+- All lookup requests **must** use ID to find the right identity.
+- Listing entities/groups by ID returns all duplicates instead of a specific,
```

Comment:
```suggestion
```
This is actually expected behavior. I originally included it to highlight the inconsistency between different list methods but it was confusing to other reviewers already so I suggest we just remove it.